ouroboros-network/test/Test/Mux.hs

```diff
@@ -117,12 +128,20 @@ startMuxSTM :: (Ord ptcl, Enum ptcl, Bounded ptcl, Mx.ProtocolEnum ptcl)
 startMuxSTM mpds wq rq mtu trace = do
     let ctx = MuxSTMCtx wq rq mtu trace
     resq <- atomically $ newTBQueue 10
-    void $ Mx.muxStart mpds (writeMux ctx) (readMux ctx) (sduSizeMux ctx) (return ()) (Just $ rescb resq)
+    void $ async (spawn ctx resq)
 
     return resq
   where
+    spawn ctx resq = do
```

Comment:
Does the `MuxSTMCtx` record serve much purpose anymore? Is it only used as a function arg to `queuesAsMuxBearer` now?

Reply:
That's true, I'll remove it.